src/tasks/semantic-release/index.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const fs = require('fs');
+const { MrmError, packageJson, lines, yaml, markdown, uninstall } = require('mrm-core');
+
+const PACKAGE_NAME = 'semantic-release';
+
+function hasSemanticRelease() {
+	const commands = yaml('.travis.yml').get('after_success');
+	return commands && commands.find(cmd => cmd.includes(PACKAGE_NAME)).length > 0;
+}
+
+module.exports = function(config) {
+	// Require .travis.yml
+	if (!fs.existsSync('.travis.yml')) {
+		throw new MrmError(
+			`Run travis task first:
+  mrm travis`
+		);
+	}
+
+	// package.json
+	const pkg = packageJson('package.json');
+
+	if (!hasSemanticRelease()) {
+		throw new MrmError(
+			`Setup semantic-release first:
+  npm i -g semantic-release-cli@latest && semantic-release-cli setup
+  
+semantic-release needs to add required auth keys to Travis CI.
+WARNING: Do not agree to update your .travis.yml when asked.
+
+More info:
+https://github.com/semantic-release/semantic-release#setup
+`
+		);
+	}
+
+	// Remove semantic-release devDependency
+	if (pkg.get(`devDependencies.${PACKAGE_NAME}`)) {
+		pkg.unset(`devDependencies.${PACKAGE_NAME}`);
+	}
+
+	// Remove semantic-release script
+	if (pkg.getScript(PACKAGE_NAME)) {
+		pkg.removeScript(PACKAGE_NAME);
+	}
+
+	// Save package.json
+	pkg.save();
+
+	yaml('.travis.yml')
+		// Remove the official semantic-release runner from commands list on .travis.yml
+		.set(
+			'after_success',
+			yaml('.travis.yml').get('after_success').filter(cmd => cmd !== 'npm run semantic-release')
```

Comment:
I think it would be better to store `yaml('.travis.yml')` in a variable instead of reading the file several times. At least code would be more readable.